mysql-test/t/all_persisted_variables.test

```diff
@@ -73,7 +76,10 @@ performance_schema.global_variables WHERE variable_name NOT IN
 'innodb_monitor_reset',
 'innodb_monitor_reset_all',
 'innodb_default_encryption_key_id',
-'rbr_exec_mode');
+'rbr_exec_mode',
+'pxc_maint_mode',
+'pxc_strict_mode',
+'wsrep_trx_fragment_size');
```

Comment:
`Query 'SET GLOBAL $var_name = @@global.$var_name' failed with wrong error 1105: 'pxc_maint_mode can be changed only if node is cluster-node'`
1105 is ER_UNKNOWN_ERROR. I don't want to add it to the list of accepted errors as it could suppress too much and also don't want to change current software behavior.

Similar situation for two other variables

`Query 'SET GLOBAL $var_name = @@global.$var_name' failed with wrong error 1105: 'pxc_strict_mode can be changed only if node is cluster-node',`